addons/core/functions/fnc_haveSWRadio.sqf

```diff
@@ -31,6 +31,13 @@ if (isNil "TFAR_currentUnit" || {isNull (TFAR_currentUnit)}) exitWith {false};
     true;
 } count (assignedItems TFAR_currentUnit);
 
+if (player != TFAR_currentUnit) then {
```

Comment:
If _result is already true we don't need to do this. Also this would return true if `TFAR_currentUnit` has a Radio but `player` doesn't. Not sure if this is expected behaviour.